src/cli/README.md

```diff
@@ -1063,6 +1063,17 @@ The parameters after `hostname` are optional. Any unspecified (or zero) value fo
 > DNS response for ipv6.google.com - 2a00:1450:401b:801:0:0:0:200e TTL: 300
 ```
 
+The DNS server IP can be an IPv4 address, which will be synthesized to an IPv6 address using the preferred NAT64 prefix from the network data.
+
+> Note: The command will return `InvalidState` when the DNS server IP is an IPv4 address and the preferred NAT64 prefix is unavailable.
```

Comment:
nit: s/**and** the preferred NAT64 prefix is unavailable/**but** the preferred NAT64 prefix is unavailable

Here and below